packages/core/src/internal/clients/leaderboard/AbstractLeaderboard.ts

```diff
@@ -38,10 +38,10 @@ export abstract class AbstractLeaderboard implements ILeaderboard {
    * {@link LeaderboardUpsert.Success} on success.
    * {@link LeaderboardUpsert.Error} on failure.
    */
-  public async leaderboardUpsert(
-    elements: Map<number, number>
+  public async upsert(
```

Comment:
Public classes are in AbstractLeaderboard.ts
These are all inherited by the `Leaderboard` class in each package.

Reply:
ooohhhh right and the PreviewClient classes just have `.leaderboard()`. Okay you're right, my bad.